packages/esm-patient-chart-app/src/routes.json

```diff
@@ -13,6 +13,12 @@
       "online": true,
       "offline": true
     },
+    {
+      "name": "encounter-list-table-tabs",
```

Comment:
Since this is the name people will need to add to their actual configs, it would be better if we were using a more understandable name like `encounter-list` or `encounter-table` (my preference)